### DISCUSSION TITLE
ToggleSwitch styling for the Off mode inconsistent with Material standards

### DISCUSSION THREAD

**Discussion by LilithSilver** [AvaloniaCommunity/Material.Avalonia#378] [General]
Material standards:

![unnamed](https://github.com/AvaloniaCommunity/Material.Avalonia/assets/84940819/841f4b1b-16a4-4cc5-ac32-734edf0f9a18)

Material.Avalonia, from the demo:

![image](https://github.com/AvaloniaCommunity/Material.Avalonia/assets/84940819/de222cb1-882b-49fa-873b-4e71ddc536ae)

Note in particular the off-switch and disabled off-switch. 

For dark mode, I couldn't find a design online, but it doesn't look right to me either:
![image](https://github.com/AvaloniaCommunity/Material.Avalonia/assets/84940819/d8d4954b-c308-4237-a091-463a315d6710)

Is this intentional? I can submit a proper issue if not.

**Comment by LilithSilver** [comment]
Another thing I realized -- it looks like the underlying switch background is using some sort of filtered overlay instead of another version of the primary/accent color. That can lead to some bad colors. (I can't find the specification for material on that point though, so I might be wrong).

**Comment by SKProCH** [comment]
Yep. Seems like this is not intentionally. You can create an issue for that.
